src/sass/components/_product-box.scss

```diff
@@ -27,8 +27,25 @@
     }
 
     .buttons {
-      display: flex;
       justify-content: space-between;
+      display: none;
+    }
+  }
+
+  &:hover {
+    .buttons {
+      display: flex;
+    }
+
+    .photo {
+      padding: 80% 10px 0 10px;
+    }
+
+    .price {
+      .btn-main-small {
+        background-color: $primary;
+        cursor: pointer;
```

Comment:
Jedynie tu się zastanawiam, czy faktycznie ustawiać zmianę kursora po najechaniu na `.photo` czy może ujednolicić to z pozostałymi elementami `.btn-main-small` i zmienić `div` na `a`... ALE UWAGA! Sama ta zmiana nie wystarczy ;)